projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php

```diff
@@ -52,19 +52,18 @@ public static function wp_dashboard_setup() {
 			return;
 		}
 
-		if ( Jetpack::is_connection_ready() ) {
-			add_action( 'jetpack_dashboard_widget', array( __CLASS__, 'dashboard_widget_footer' ), 999 );
-		}
+		// TODO: Migrate this head function into this class.
+		add_action( 'admin_head', 'stats_dashboard_head' );
```

Comment:
I think this might need to go in the block testing for `is_connection_ready()` starting on line 58.

Reply:
Good catch; the stats module doesn't seem to be loaded unless there's a connection.